src/modules/Dashboard/features/Applet/Schedule/Calendar/Calendar.styles.ts

```diff
@@ -422,19 +409,15 @@ export const StyledCalendarWrapper = styled(Box, shouldForwardProp)`
     }
 
     .rbc-event {
-      max-width: 92%;
-      margin: 0 auto;
-      transition: ${variables.transitions.opacity};
-
-      &:hover {
-        opacity: 0.9;
-      }
-
       &:focus {
-        outline-color: ${variables.palette.primary};
+        outline: none;
       }
     }
 
+    .rbc-row-segment {
+      padding: ${theme.spacing(0, 0, 0.18, 0)};
```

Comment:
does it make sense having decimal part of pixel (1rem = 10px so 0.18rem = 1.8 px)?

Reply:
yes:)